### PR TITLE
Made it possible to paint windows to make them opaque.

### PR DESCRIPTION
**Pull request by xxalpha**
Edit: Changed it so only darker colors make the window opaque.
Changed only the colour var.
Painting or cleaning tinted windows will not make them see through.

-------------

My first PR.

scrnshot1

Paint them with spray cans, clean them with soap.

Also renamed two vars to be less confusing.

### CODE REVIEW

**Comment by francinum** [comment]
>Color instead of Colour
:-1:

**Comment by Anonus** [comment]
Should work with paint cans too.
>the color/colour wars
stop it

**Comment by RemieRichards** [comment]
changing ```colourName``` to ```colorName``` was done 1000% needless.

**Comment by xxalpha** [comment]
I wouldn't call consistency needless.

**Comment by RemieRichards** [comment]
We've been over this, we're not doing the whole "Unified spelling" change.
We're continuing as we always have, Developers write in what they prefer and DON'T NEEDLESSLY CHANGE THINGS.

**Comment by xxalpha** [comment]
I am not scouring the codebase for 'colour', I changed the two vars from the same type of object because one was a bad name. I don't write 'colour', so it would have 'paint_color' and 'colourName', so I just changed both.

**Comment by RemieRichards** [comment]
You could have just as easily written ```paint_colour``` then you'd have had your sweet "consistency" with LESS WORK TO DO.

**Comment by xxalpha** [comment]
Stop bullying my PR, remie.

**Comment by GunHog** [comment]
Inconsistent spellings are a consistency and maintainability issue. US or UK spelling, I suggest we pick one and stick with it, or this problem just gets worse.

**Comment by RemieRichards** [comment]
We've dealt with it for 5 years, it's literally a non issue.
Just stop changing shit you didn't write FOR NO PURPOSE.

**Comment by xxalpha** [comment]
Not compatible with brits.

**Comment by Anonus** [comment]
Do you mind if I just take this and do it myself?

**Comment by Cheridan** [comment]
omg stop nerding out over fucking spelling who cares either way

**Comment by duncathan** [comment]
@RemieRichards look it doesn't matter which spelling is used and if this PR were just changing it for the sake of making it "correct" I'd be against it but
`colour` as a var name is disgusting and confusing; `paint_color` is perfectly fine and descriptive

**Comment by RemieRichards** [comment]
@duncathan Omg, I'm not defending ```colour```, I'm having a go at xxalpha for renaming a completely seperate var, rather than just having his new var name be ```paint_colour``` instead of ```paint_color```+```colorName```

tl;dr he needlessly changed 2 vars vs. changing 1 var.

**Comment by duncathan** [comment]
@RemieRichards ohh I see your point
Well for this I'll just say "it's not really a big enough deal to make a fuss about"
I will say that given the fact that `color` is used internally it'd be nice to at the very least maintain consistency with that but again, doesn't really matter too much

File: code/game/objects/items/weapons/clown_items.dm [hunk 1]
```diff
@@ -66,6 +66,10 @@
 	else if(ishuman(target) && user.zone_sel && user.zone_sel.selecting == "mouth")
 		user.visible_message("<span class='warning'>\the [user] washes \the [target]'s mouth out with [src.name]!</span>", "<span class='notice'>You wash \the [target]'s mouth out with [src.name]!</span>") //washes mouth out with soap sounds better than 'the soap' here
 		return
+	else if(istype(target, /obj/structure/window))
+		user << "<span class='notice'>You clean \the [target.name].</span>"
+		target.color = initial(target.color)
+		target.SetOpacity(0)
```
Comment (Anonus) [inline code/game/objects/items/weapons/clown_items.dm]: This will cause tinted windows to become non-opaque, you need to check if it's tinted.

**Comment by WJohn** [comment]
:+1: 
Cute.

File: code/game/objects/items/weapons/clown_items.dm (hunk 1 above)
Comment (xxalpha) [inline code/game/objects/items/weapons/clown_items.dm]: Tinted windows aren't used.

Comment (francinum) [inline code/game/objects/items/weapons/clown_items.dm]: @xxalpha Yes they are.

Comment (duncathan) [inline code/game/objects/items/weapons/clown_items.dm]: They shouldn't be tbh

**Comment by Jordie0608** [comment]
Painting windows is cool, opaque windows aren't.

File: code/game/objects/items/weapons/clown_items.dm (hunk 1 above)
Comment (francinum) [inline code/game/objects/items/weapons/clown_items.dm]: They are, mostly for the chapel confession rooms, but some mapper probably used them elsewhere. Anyhow, that's not an excuse to not add a check for them.

**Comment by Niknakflak** [comment]
Watching remie freak out over the color thing again is funny.  If it doesn't "matter", who cares?

File: code/game/objects/items/weapons/clown_items.dm (hunk 1 above)
Comment (Saegrimr) [inline code/game/objects/items/weapons/clown_items.dm]: Tinted windows are in box maint as well below toxins. It's been there for ages if anybody here has even been in maint in the last 2 years.

**Comment by optimumtact** [comment]
I like the idea, @Jordie0608 what is so wrong with opaque windows?

**Comment by Niknakflak** [comment]
Watching remie freak out over the color thing again is funny. If it doesn't "matter", who cares?

**Comment by optimumtact** [comment]
Taking passive aggressive potshots against people isn't funny. Keep your little back and forth off github please.

**Comment by CosmicScientist** [comment]
Painting windows, :+1:.

Lumping it together with changing var names that apply to more than the title of this PR, :-1:.

**Comment by ike709** [comment]
Window painting: :+1: 

Reenacting the War of 1812: :-1:
